src/http-servers/echo-server.ts

```diff
@@ -0,0 +1,7 @@
+import http from "http";
+
+http.createServer()
+  .on("request", (req, res) => {
+    req.pipe(res);
```

Comment:
This server will do actually nothing. Please, consume `message` query param and when the user sends response with it like `localhost:3000?message=hello` he should be responded with plain text and should see `hello` on the screen